### DISCUSSION TITLE
Remove WeekSubscribe extension method

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [xamarin/XamarinCommunityToolkit#1001] [Ideas]
As discovered in https://github.com/xamarin/XamarinCommunityToolkit/pull/987 `WeekSubscribe` should never be used on week events. If it does, subscriber will get unsubscribed on first GC run.

And since this repository also includes `ObservableObject` that uses `WeekEventManager`, it makes `WeekSubscribe` obsolete. Both do the same thing, and if used at the same time, user will get issue descried above.

So, should it be removed, or should we add a check that subscriber cannot be `ObservableObject`?

CC: @pictos, @brminnick, @jfversluis

**Comment by pictos** [comment]
To me it's ok to remove it.

**Reply by TheCodeTraveler** [reply]
Agreed
